mps_youtube/commands/__init__.py

```diff
@@ -11,10 +11,22 @@
 RS = r'(?:(?:repeat|shuffle|-[avfw])\s*)'
 PL = r'\S*?((?:RD|PL|LL)[-_0-9a-zA-Z]+)\s*'
 
-
-def command(regex):
+## @command decorator
+##
+## The @command decorator takes two arguments, one is a regex that
```

Comment:
It's not very important, but I wouldn't describe a polymorphic function as taking "two arguments"

Reply:
Yeah, that was kind of embarrassingly bad.
I've updated it.